tests/filtertransformers/test_mongo.py

```diff
@@ -259,46 +261,74 @@ def test_operators(self):
         with pytest.raises(VisitError):
             self.transform('"some string" > "some other string"')
 
-    def test_filtering_on_relationships(self):
+    def test_filtering_on_relationships(self, mapper):
         """Test the nested properties with special names
         like "structures", "references" etc. are applied
         to the relationships field.
 
         """
+        from optimade.filtertransformers.mongo import MongoTransformer
+
+        t = MongoTransformer(mapper=mapper("StructureMapper"))
+        p = LarkParser(version=self.version, variant=self.variant)
```

Comment:
The line p = LarkParser(version=self.version, variant=self.variant) is repeated many times through the test. Why do you not declare it once ?

Reply:
It's declared once per test (each method here is a separate test that runs (ideally) in its separate context).

Reply:
It is a bit annoying, we could define a module scoped `parser` pytest fixture in `conftest.py` but this may be marginally clearer.